### PR TITLE
fixes #1743 Corrected typos and Grammatical errors

### LINKED ISSUE #1743: Typo and Grammatical errors in README.md
## Typo errors
"avaliable" expected to be =>"available"
"performace" expected to be =>"performance"
"recieve" expected to be =>"receive"
"succesfully" expected to be =>"successfully"

and also **some grammatical errors** must be corrected

**I would like to fix this issue**

### PR DESCRIPTION
**Pull request by girijamanojkumarreddy**
Corrected typos and Grammatical errors as mentioned in issue #1743

Fixed #1743 

## Changes: 
"avaliable" changed to =>"available"
"performace" changed to =>"performance"
"recieve" changed to =>receive"
"succesfully" changed to =>"successfully"

Did some minor grammatical changes which enhance the readability of the file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,7 +57,7 @@ Do you have many different apps in your Android Phone for the purpose of clickin
 
 * Finally, after all this editing you can easily share the image to your favourite social media sites with our easy-to-use sharing feature.
 
-* Facebook, Twitter, NextCloud, OwnCloud, Imgur, Dropbox, Box, Flickr, Pinterest, Instagram, Whatsapp and Tumblr - You name it and we have it covered.
+* Facebook, Twitter, NextCloud, OwnCloud, Imgur, Dropbox, Box, Flickr, Pinterest, Instagram, Whatsapp, and Tumblr - You name it and we have it covered.
```
Comment (sachincool) [inline README.md]: Remove the `,` before `and` that is not necessary.
https://www.grammarly.com/blog/comma-before-and/  this might help

Comment (girijamanojkumarreddy) [reply README.md]: Though it is not necessary, I thought it will be perfect if we use "Oxford commas". :)

Comment (sachincool) [reply README.md]: cool, i'll remove it then.

File: README.md [hunk 2]
```diff
@@ -32,20 +32,20 @@ Phimp.me is a Photo App for Android that aims to replace proprietary photo appli
...
 * You can use your voice actions to invoke Camera, Just say "Ok Google click a picture". You can also select front and rear camera based on voice.
 
-* Browse the local gallery inside the app with folder and all photos mode. Copy, move and add description to the images.
+* Browse the local gallery inside the app with folder and all photos mode. Copy, move and add a description to the images.
 
-* Edit images with various cool filters with optimized performace, built using OpenCV library.
+* Edit images with various cool filters with optimized performance, built using OpenCV library.
 
-* Enhance contrast, hue, satur, temp, tint and sharpness of the image.
+* Enhance contrast, hue, satur, temp, tint, and sharpness of the image.
```
Comment (isuruAb) [inline README.md]: we don't need comma after this

**Pull request merged**